docs/rest/rest-catalog-open-api.yaml

```diff
@@ -652,7 +638,7 @@ components:
           description: HTTP response code
           example: 404
 
-    CatalogConfiguration:
+    RESTCatalogConfig:
```

Comment:
Why this rename?

Reply:
This matches the values that are used. Although now we just have `RESTCatalogConfigResponse`.